_rules/element-lang-valid-de46e4.md

```diff
@@ -97,37 +106,73 @@ The `lang` attribute value has a valid language tag, but a syntactically invalid
 
 #### Failed Example 1
 
-The `lang` attribute value is not a primary language subtag.
+The `lang` [attribute value][] is not a primary language subtag.
 
 ```html
 <html>
 	<body>
-		<article lang="dutch"></article>
+		<article lang="dutch">
+			Zij liepen een vreemde Tiki bar binnen, aan de rand van een dorpje aan het strand.
+		</article>
 	</body>
 </html>
 ```
 
 #### Failed Example 2
 
-The `lang` attribute value is not empty ("") and is not a primary language subtag.
+The `lang` [attribute value][] is not empty (`""`) and is not a primary language subtag.
 
 ```html
 <html>
 	<body>
-		<article lang="#!"></article>
+		<article lang="#!">
+			They wandered into a strange Tiki bar on the edge of the small beach town.
+		</article>
 	</body>
 </html>
 ```
 
 #### Failed Example 3
 
-The `lang` attribute value consists of only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace) and is not a [valid language tag][].
+The `lang` [attribute value][] consists of only [whitespace][] and is not a [valid language tag][].
 
 ```html
 <html>
 	<body>
-		<article lang=" ">
-			The quick brown fox jumped over the lazy dog
+		<article lang="  ">
+			They wandered into a strange Tiki bar on the edge of the small beach town.
+		</article>
+	</body>
+</html>
+```
+
+#### Failed Example 4
+
+The `lang` [attribute value][] does not have a valid language tag, and its [descendant][] is not [included in the accessibility tree][] though it is still [visible][].
```

Comment:
Being included in the accessibility tree has nothing to do with the Expectation.
This description makes it look like this is a reason for failure. This is not the case.

If the idea of this (and following example) is to show that Applicability is a "or" (visible or accessible), I'd move them to Passed Example (updating code) with a description pointing that ("it is not included in the accessibility tree but is still visible, hence it is Applicable").